### ISSUE TITLE
Gradle Deprecation Warning for ConfigureUtil

### ISSUE THREAD

**Issue by ArvinB** [OpenLiberty/ci.gradle#900]
The OpenLiberty Gradle plugin is throwing a deprecation warning for the use of `ConfigureUtil` located [here](https://github.com/OpenLiberty/ci.gradle/blob/main/src/main/groovy/io/openliberty/tools/gradle/extensions/ServerExtension.groovy#L65-L95). More information on how to correct this is [here](https://docs.gradle.org/current/javadoc/org/gradle/util/ConfigureUtil.html).

Deprecation Warning from Gradle:

```
> Configure project :
The org.gradle.util.ConfigureUtil type has been deprecated. This is scheduled to be removed in Gradle 9.0. Consult the upgrading guide for further information: https://docs.gradle.org/8.9/userguide/upgrading_version_8.html#org_gradle_util_reports_deprecations
        at build_f5brw61kq6jdqi9ls2l1po8ur$_run_closure8.doCall$original(/Users/cakasky/Documents/GitHub/wca-code-explanation/build.gradle:125)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
        at build_f5brw61kq6jdqi9ls2l1po8ur.run(/Users/cakasky/Documents/GitHub/wca-code-explanation/build.gradle:124)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
```

From `build.gradle`

```
liberty {
    server {
        name = project.server.name
        deploy.apps = [war]
        configDirectory = file("src/main/liberty/config")
        mergeServerEnv = true
    }
}
```

Related issue:
https://github.com/OpenLiberty/ci.gradle/issues/868

**Comment by ArvinB** [comment]
@cherylking Can I assume this is resolved with the latest plugin?

**Comment by cherylking** [comment]
It will be when we publish the next release @ArvinB

**Comment by cherylking** [comment]
Targeting end of next week to publish the next release `3.9`.

**Comment by cherylking** [comment]
This is fixed in `3.9.1` release that was published yesterday.
